CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+###
+- Joplin CGE datasets [#38](https://github.com/IN-CORE/incore-data-release/issues/38)
```

Comment:
Those datasets are actually for the Housing household recovery sequential analysis